stable/memcached/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 name: memcached
-version: 1.2.1
-description: Free & open source, high-performance, distributed memory object caching system.
+version: 2.0.0
```

Comment:
Please update only the minor version

Reply:
ignoring this comment as it has been agreed to update Major version for breaking change